CryptoPrices/Sources/Domain/Tests/UseCasesTests/MyCoinsUseCaseTests.swift

```diff
@@ -0,0 +1,5 @@
+import XCTest
+@testable import UseCases
+
+final class MyCoinsUseCaseTests: XCTestCase {
+}
```

Comment:
@minhnimble Since we don't have unit tests (yet) I didn't add the `Quick/Nimble`. But I totally agree with you that this task is like a base one, to make (or at least try to make) everything ready for upcoming tasks. Will update now 🙏